pkg/chunk/cache/background.go

```diff
@@ -80,21 +80,33 @@ func (c *backgroundCache) Stop() {
 	c.Cache.Stop()
 }
 
+const keysPerBatch = 100
+
 // Store writes keys for the cache in the background.
 func (c *backgroundCache) Store(ctx context.Context, keys []string, bufs [][]byte) {
```

Comment:
Do we have a test convering the case the input keys are > `keysPerBatch`?

Reply:
I modified the test to ensure this.